Base/Python/slicer/util.py

```diff
@@ -156,7 +156,7 @@ def showStatusMessage(message, duration = 0):
   mw.statusBar().showMessage(message, duration)
   return True
 
-def findChildren(widget=None, name="", text="", title="", className=""):
```

Comment:
Nice, thanks for the suggestion, that works well. See fe3669f
I removed the toolTip commit.